deprecate.dd

```diff
@@ -814,8 +814,8 @@ $(H4 Rationale)
 )
 
 Macros:
-	DEPLINK=$(RELATIVE_LINK2 $0, $0)
+	DEPLINK=$(RELATIVE_LINK2 $1, $2)
 	DEPLINK2=$(LINK2 $1.html#$2, $2)
-	DEPNAME=$(LNAME2 $0, $0)
+	DEPNAME=<a name="$1">$(LNAME2 $2, $1)
```

Comment:
Need to close the `<a>`.

Can we use the same order of arguments as with DEPLINK?